source/SIL.AppBuilder.Portal.Frontend/package.json

```diff
@@ -29,6 +29,7 @@
     "auth0-js": "^9.6.1",
     "auth0-lock": "^11.7.2",
     "jwt-decode": "^2.2.0",
+    "pretty-ms": "^3.2.0",
```

Comment:
what's this do?

Reply:
prettyMs(1337000000);
//=> '15d 11h 23m 20s'